libs/sdk-ui-dashboard/src/model/store/backendCapabilities/backendCapabilitiesSelectors.ts

```diff
@@ -18,3 +18,23 @@ export const selectBackendCapabilities = createSelector(selectSelf, (state) => {
 
     return state.backendCapabilities!;
 });
+
+/**
+ * Returns if capability supportsKpiWidget is supported
```

Comment:
This statement looks a bit weird, what about "Does backend support kpi widgets?", or something more human friendly :)  Or link to the backend capabilities interface, where it's described . Same applies for the other capabilities selectors.